tests/notebooks/modin/MIMICHealthcareDemo.ipynb

```diff
@@ -392,10 +392,17 @@
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Can you double check `TimeSeriesTesting.ipynb`? We may have some improvements there too.

Reply:
That notebook is blocked by a features that are not related to timedelta:

- [DatetimeIndex.tz_localize](https://snowflakecomputing.atlassian.net/browse/SNOW-1017838?atlOrigin=eyJpIjoiMjZiMmY0MDQ0MTIwNGU0OWIyMzdjMWQ3ZDQwNTFkMDciLCJwIjoiaiJ9)
- [DatetimeIndex.tz_convert](https://snowflakecomputing.atlassian.net/browse/SNOW-1017838?atlOrigin=eyJpIjoiMjZiMmY0MDQ0MTIwNGU0OWIyMzdjMWQ3ZDQwNTFkMDciLCJwIjoiaiJ9)


Also:
- have to implement Timedelta tracking through `SnowflakeQueryCompiler::set_index_from_columns` (SNOW-1620436) to support the use case [here](https://github.com/snowflakedb/snowpark-python/blob/0a9bbc7a3e067cb97eac656453ac9e7ef733157d/tests/notebooks/modin/TimeSeriesTesting.ipynb#L265)

Reply:
@sfc-gh-mvashishtha Naresh's TimedeltaIndex PR supports that https://github.com/snowflakedb/snowpark-python/pull/2160

Reply:
okay, I'll revisit that part after #2160 has been merged